### ISSUE TITLE
DB name in config ignored: always uses 'dxcluster'

### ISSUE THREAD

**Issue by manolab** [coulisse/spiderweb#31]
Even if in the config file there is:

```
{
    "mysql": {
        "host": "localhost",
        "user": "myuser",
        "passwd": "mypassword",
        "db": "cluster"
    },
[...]

```
the webapp uses `dxcluster` as DB name.

From the logs in fact:

`[ERROR] (qry.py) 1142 (42000): SELECT command denied to user 'spider'@'localhost' for table `**dxcluster**`.`spot``

I had to modify some lines in `webapp.py `to hardcode there my DB name.
Can you use the DB name from the config file?

Here's the diff:

```
diff --git a/webapp.py b/webapp.py
index 9f99a2a..26a3b5a 100644
--- a/webapp.py
+++ b/webapp.py
@@ -77,14 +77,14 @@ def query_build_callsign(callsign):
     query_string = ""
     if len(callsign) <= 14:
         query_string = (
-            "(SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from dxcluster.spot WHERE spotter='"
+            "(SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from cluster.spot WHERE spotter='"
             + callsign
             + "'"
         )
         query_string += " ORDER BY rowid desc limit 10)"
         query_string += " UNION "
         query_string += (
-            "(SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from dxcluster.spot WHERE spotcall='"
+            "(SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from cluster.spot WHERE spotcall='"
             + callsign
             + "'"
         )
@@ -175,7 +175,7 @@ def query_build():
             last_rowid = 0
 
         query_string = (
-            "SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from dxcluster.spot WHERE rowid > "
+            "SELECT rowid, spotter AS de, freq, spotcall AS dx, comment AS comm, time, spotdxcc from cluster.spot WHERE rowid > "
             + last_rowid
         )
```

**Comment by coulisse** [comment]
Ok I'll check it
Thanks
Corrado

**Comment by coulisse** [comment]
Fixed with v2.4.2 version
